crates/macro-support/src/codegen.rs

```diff
@@ -1145,6 +1145,16 @@ impl ToTokens for ast::ImportType {
                         unsafe { &*(val as *const JsValue as *const #rust_name) }
                     }
                 }
+
+                #[automatically_derived]
+                impl TryFromJsValue for #rust_name {
```

Comment:
I think this can be done as a blanket implementation in `impls.rs` instead of individual codegens.